### DISCUSSION TITLE
working itunes for iphone 12?

### DISCUSSION THREAD

**Discussion by alpoman** [airsdk/Adobe-Runtime-Support#740] [Q&A]
Hello, I can't seem to get my new phone to be recognized in adobe animate for publishing.  My old iphone 6 works no problem, but my new iphone 12 with ios 14.4 wont show on refresh.  I used to use an older version of itunes to get animate to recognize my iphone 6 (it was itunes 12.1.3.6 64 bit) , but wont work on new phone.  Long story short, what is anyone using for itunes and latest animate version to publish?  Sorry, for the long winded question, huge thanks in advance to anyone who can help me,

Andrew

**Comment by FliplineStudios** [comment]
We had a similar problem with iPhone 12, not using Animate but with installing an adhoc version using the old iTunes 12.6.5.3 on Windows (the separate version Apple made which still let you install apps, unlike the newer versions).  The iPhone 12 used to install apps fine this way, but once it updated to iOS 14.3 it suddenly wouldn't appear in that old iTunes anymore for syncing/installing, though older devices like an iPhone X on 14.4 and a iPad Pro 11 on 14.4 can both still sync fine with it.

I eventually just had to use "adt" on the command line for AIR to install on the device using -installApp, just for that iPhone 12.  I can still use that old iTunes for all of the other devices even though they're all using the same version of iOS though.
